.github/workflows/e2e_test.yml

```diff
@@ -59,7 +60,7 @@ jobs:
         osc_region: ${{ secrets.OSC_REGION }}
         image_id: ${{ secrets.OMI_ID }}
         rke_version: "v1.4.2"
-        kubernetes_version: "v1.23.15-rancher1-1"
+        kubernetes_version: "v1.24.9-rancher1-1"
```

Comment:
You are upgrading in 1.25 but rancher is stuck at kubernetes 1.24 (and current release of kubernetes is 1.26).

Reply:
Yes, between 1.24 and 1.25, the API for the CSI is untouched.

And yes, rancher does not have a 1.25 kubernetes cluster => stay on 1.24